app/src/main/java/org/mifos/mobile/ui/fragments/Account.kt

```diff
@@ -0,0 +1,19 @@
+package org.mifos.mobile.ui.fragments
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.mifos.mobile.R
+import org.mifos.mobile.ui.fragments.base.BaseFragment
+
+class Account : BaseFragment() {
```

Comment:
use better class names. suffix with `Fragment`, it's a thumb of rule. refer existing code.

Reply:
okay I'll follow this naming convention.